ui/src/components/Header.tsx

```diff
@@ -28,7 +28,11 @@ import wordmark from "@/public/wordmark/wordmark-white.svg";
 import { displayDesktopOnly, displayMobileOnly } from "@/styles/responsive";
 import { analytics } from "@/utils/analytics";
 
-export default function Header() {
+export default function Header({
+  onBrandClick,
+}: {
+  onBrandClick?: () => void;
+}) {
```

Comment:
Not important for this PR, just a note:

I notice there's a bit of prop drilling here, nothing too bad but a good sign that at some point we might want to move the drawer open/close state into some state store (e.g. Zustand or Jotai) so that we can toggle it without all the extra plumbing

Reply:
I can make a new issue for this, I think I have a related draft issue in the project board

Reply:
Yeah, the drawer's probably a good candidate for that